float64 slip_spring
float64 standing_leg
float64 hop_height
float64 slip_leg

uint8 main_controller
uint8 spring_type
uint8 force_type
uint8 stance_controller
uint8 hop_type
float64 hopping_state

# Event Trigger Options
float64 force_threshold_td
float64 force_threshold_to
float64 position_threshold_td

# Leg Gains
float64 leg_pos_kp
float64 leg_for_kp
float64 leg_pos_kd
float64 leg_for_kd

# Hip Gains
float64 hip_pos_kp
float64 hip_pos_kd
float64 left_toe_pos
float64 right_toe_pos
